### PR TITLE
Themes (cupertino): Set individual selectors

### PR DESCRIPTION
**Pull request by jonaaix**
Fixed theme selectors as suggested

### CODE REVIEW

File: docs/plugins/theme-cupertino-dark/manifest.json [hunk 1]
```diff
@@ -1,21 +1,21 @@
 {
   "version": "1.0.0",
   "manifestVersion": "0.1.0",
-  "name": "theme-cupertino-dark",
+  "name": "theme-cupertino-light",
```
Comment (antelle) [inline docs/plugins/theme-cupertino-dark/manifest.json:4]: This doesn't look right, dark theme is called `light`

File: docs/plugins/theme-cupertino-dark/manifest.json [hunk 2]
```diff
@@ -1,21 +1,21 @@
...
     "email": "btxtiger@outlook.com",
     "url": "https://github.com/btxtiger"
   },
   "resources": {
-    "css": "aT7cFpD9SiWopZiH63gQSOZFoD8aLxBEdbZuK+Y3YzYsaS79uDqm7ZNNdNszB4uu9eDrtZfyUM6BiL50Y9CmXICDODLDb5GQodZbzLPaqNN5K8JF2pIAAHkY+qGZsStxMaQCm3Tq3bX+MuEWtSF2tyr+BN2nPcrqBrNyK/GlrdzAVuK8NXTZWuuh1MfVWUbGGdlhOoNI0n/TlSHv+Miqtycx/MVSEkPLDMcg7eiIq63i7GryeqcuCHpepwQeX1YL+v5aPqsnquf/A5l+VTC+7yg6GvmM3Nwjwx+IEh86jdbH11z+ZXdqd6bFEKcLn6IHmzI2yfrpFNnQQzZSnq+lQA=="
+    "css": true
   },
-  "url": "https://plugins.keeweb.info/plugins/theme-cupertino-dark",
-  "publicKey": "MIIBIjANBgkqhkiG9w0BAQEFAAOCAQ8AMIIBCgKCAQEAnRq2k3TTx0ewTe6wDr6QVeB5diwiIWzsJD+ApfZu1KNPedcAgslAfjpNsYF1if6cYsPMJH70xJ2np6RQBl1VPdwShOuxkD7m0BD5Hw/Aar8Hdp5cvAdOOMdBO+0DbGeUMy+z66s+oUCJmqVp19T6PkkxbhN08rgtT7v+aFvrbqbO/vlsskbJpH2K2io+e1XmRGPnSr9q4KSqfGbTfe5gLwDIOFd66Z4mb5Utb5wWpsy6Gjh06Yf257AccGD3A1bkTNOyeeX0tqciYBePWMk0icP/aZ6hnErfhnUKf3tOgPLppSHiGcaSKekhChZ2xLUs3U64JwrXSmwHj+TzdO3S0QIDAQAB",
+  "url": "https://github.com/btxtiger/keeweb-cupertino-theme",
```
Comment (antelle) [inline docs/plugins/theme-cupertino-dark/manifest.json:14]: please revert this change

File: docs/plugins/theme-cupertino-dark/manifest.json [hunk 3]
```diff
@@ -1,21 +1,21 @@
...
+  "publicKey": "MIIBIjANBgkqhkiG9w0BAQEFAAOCAQ8AMIIBCgKCAQEAlRC2zOmBL8RUOoXRkEdpkb/Le3y/VKC+4mGefYxg9a3KO4Jeqayt1Xm38/QNLMTz7Fy8D3N+BaOLN2Avky/m5ggmvynHieW+rxEGMwWozh4aAB3l9JO8BXm0+98jME+TjnbwS81QM985Y3PUhQaTCNqkWeRX02CHZQWSU+BY+XVEr37f0s35sELZWEMvxf4KKp/oagOdVS7FFuprsMV/e+q4ezOpmx7o6ZVeEhfC0rVB4CxQZ7jeY3RUjiVzI1vT5D0tuxxniDN2ZKdG4J4uIE7A4cLkVFTNUl0EHIjmxIkMqUX+fyUgrpDv9KUd5d4UeqErMX3zGPuKferGIGZPXQIDAQAB",
```
Comment (antelle) [inline docs/plugins/theme-cupertino-dark/manifest.json:15]: and this one too

File: docs/plugins/theme-cupertino-light/manifest.json [hunk 4]
```diff
@@ -1,21 +1,21 @@
 {
   "version": "1.0.0",
   "manifestVersion": "0.1.0",
-  "name": "theme-cupertino-light",
+  "name": "theme-cupertino-dark",
```
Comment (antelle) [inline docs/plugins/theme-cupertino-light/manifest.json:4]: Also doesn't look right, light theme is called dark

**Comment by jonaaix** [comment]
Hawk's eyes 😄 👍 
So the public key should always remain the previously set? @antelle

**Comment by jonaaix** [comment]
I'm gonna create a new PR, so it wouldn't mess up the history.

**Comment by antelle** [comment]
Yes, right now 3rd-party public keys are not accepted.
Previously the plugin repo was designed to use different public keys, however I quickly realized that this way we'll open a very easy door to exploiting this.

**Pull request closed without merging**